### PR TITLE
UI plan

### PR DESCRIPTION
**Pull request by kimyu-ng**

### CODE REVIEW

File: plans/kn6254_thv92.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+});
+```
+
+
+* We are looking for UI enhancement and an intuitive design of Card on the page.
+
+
+## Goal
+
+* We want to find out the reason and fix the issue of slow zooming in or zooming out on open layers when loading actual layer from geoserver.
```
Comment (mmccartney) [inline plans/kn6254_thv92.md:44]: i need help understanding the problem being solved here.  is open layers slower than leaflet?  how are you measuring performance?  do you have specific performance goals (e.g. latency < 200ms) ?

File: plans/kn6254_thv92.md [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+
+```javascript
+var wmsLayer = new ol.layer.Tile({
+               		....
+               		params: {LAYERS: 'basic'} //QuickStart //when using the actual layer, the layer get screwed up
+               	});
+```
+
+* We would like to change the stylesheet of the site.
```
Comment (mmccartney) [inline plans/kn6254_thv92.md:53]: why?

File: plans/kn6254_thv92.md (hunk 1 above)
Comment (kimyu-ng) [inline plans/kn6254_thv92.md:44]: I believe openlayers is not any slower than leaflet. We're trying to port from leaflet to openlayers. However, we don't have deep understanding on how the technology works. So there might be some issue on our current porting

File: plans/kn6254_thv92.md (hunk 2 above)
Comment (kimyu-ng) [inline plans/kn6254_thv92.md:53]: Quickstart is the layer created in geoserver running on Rackspace. However, when we tried to substitute the layer with any non-existence dummy layer, the zooming is ridiculously fast. It is possibly related to the current implementation of UI.

**Pull request merged**